adal/src/telemetry/java/com/microsoft/aad/adal/EventStrings.java

```diff
@@ -24,139 +24,149 @@
+    static final String TOKEN_TYPE_IS_RT = EVENT_PREFIX + "is_rt";
 
-    static final String BROKER_VERSION = "Microsoft.ADAL.broker_version";
+    static final String TOKEN_TYPE_IS_MRRT = EVENT_PREFIX + "is_mrrt";
 
-    static final String BROKER_APP_USED = "Microsoft.ADAL.broker_app_used";
+    static final String TOKEN_TYPE_IS_FRT = EVENT_PREFIX + "is_frt";
 
-    static final String TOKEN_TYPE = "Microsoft.ADAL.token_type";
+    static final String TOKEN_TYPE_RT = EVENT_PREFIX + "rt"; // Android only
 
-    static final String TOKEN_TYPE_IS_RT = "Microsoft.ADAL.is_rt";
+    static final String TOKEN_TYPE_MRRT = EVENT_PREFIX + "mrrt"; // Android only
 
-    static final String TOKEN_TYPE_IS_MRRT = "Microsoft.ADAL.is_mrrt";
+    static final String TOKEN_TYPE_FRT = EVENT_PREFIX + "frt"; // Android only
 
-    static final String TOKEN_TYPE_IS_FRT = "Microsoft.ADAL.is_frt";
+    static final String CACHE_EVENT_COUNT = EVENT_PREFIX + "cache_event_count";
 
-    static final String TOKEN_TYPE_RT = "Microsoft.ADAL.rt"; // Android only
+    static final String UI_EVENT_COUNT = EVENT_PREFIX + "ui_event_count";
 
-    static final String TOKEN_TYPE_MRRT = "Microsoft.ADAL.mrrt"; // Android only
+    static final String HTTP_EVENT_COUNT = EVENT_PREFIX + "http_event_count";
 
-    static final String TOKEN_TYPE_FRT = "Microsoft.ADAL.frt"; // Android only
+    static final String HTTP_PATH = EVENT_PREFIX + "http_path";
 
-    static final String CACHE_EVENT_COUNT = "Microsoft.ADAL.cache_event_count";
+    static final String HTTP_USER_AGENT = EVENT_PREFIX + "user_agent";
 
-    static final String UI_EVENT_COUNT = "Microsoft.ADAL.ui_event_count";
+    static final String HTTP_METHOD = EVENT_PREFIX + "method";
 
-    static final String HTTP_EVENT_COUNT = "Microsoft.ADAL.http_event_count";
+    static final String HTTP_METHOD_POST = EVENT_PREFIX + "post";
 
-    static final String HTTP_PATH = "Microsoft.ADAL.http_path";
+    static final String HTTP_QUERY_PARAMETERS = EVENT_PREFIX + "query_params";
 
-    static final String HTTP_USER_AGENT = "Microsoft.ADAL.user_agent";
+    static final String HTTP_RESPONSE_CODE = EVENT_PREFIX + "response_code";
 
-    static final String HTTP_METHOD = "Microsoft.ADAL.method";
+    static final String HTTP_API_VERSION = EVENT_PREFIX + "api_version";
 
-    static final String HTTP_METHOD_POST = "Microsoft.ADAL.post";
+    static final String REQUEST_ID_HEADER = EVENT_PREFIX + "x_ms_request_id";
 
-    static final String HTTP_QUERY_PARAMETERS = "Microsoft.ADAL.query_params";
+    static final String SERVER_ERROR_CODE = EVENT_PREFIX + "server_error_code";
 
-    static final String HTTP_RESPONSE_CODE = "Microsoft.ADAL.response_code";
+    static final String SERVER_SUBERROR_CODE = EVENT_PREFIX + "server_sub_error_code";
 
-    static final String HTTP_API_VERSION = "Microsoft.ADAL.api_version";
+    static final String TOKEN_AGE = EVENT_PREFIX + "token_age";
```

Comment:
I thought we agreed on Microsoft.ADAL.rt_age for this.

Reply:
Corrected 👍